controller.go

```diff
@@ -42,25 +44,27 @@ type K8SEvent struct {
 }
 
 func (c *K8SController) mainloop(item interface{}) {
-	log.Debug().Str("queue_addr", fmt.Sprintf("%p", &c.queue)).Int("queue_len", c.queue.Len()).Msg("mainloop: running from top")
```

Comment:
queue_addr is not really useful since it's ephemeral, I added this a while ago when I was debugging and learning about the repo.